lib/v2/screens/sign_up/add_phone_number/add_phone_number.dart

```diff
@@ -33,37 +35,43 @@ class _AddPhoneNumberState extends State<AddPhoneNumber> {
         body: BlocConsumer<SignupBloc, SignupState>(
           listener: (context, state) {},
           builder: (context, state) {
-            return Padding(
-              padding: const EdgeInsets.all(16),
-              child: Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: <Widget>[
-                  TextFormFieldCustom(
-                    labelText: "Add Phone Number (optional)".i18n,
-                    keyboardType: TextInputType.phone,
-                    controller: _keyController,
-                    inputFormatters: [
-                      phoneNumberInternationalFormatter,
+            return Stack(
+              children: [
+                Padding(
+                  padding: const EdgeInsets.all(16),
```

Comment:
Please use Ui constant horizontalEdgePadding (16), if we decide to change padding size latter on we only need to change it in one place